### DISCUSSION TITLE
v3.4.0

### DISCUSSION THREAD

**Discussion by mattieserver** [mattieserver/netbox-topology-views#284] [Announcements]
## Notice
This version only works for netbox 3.5.0 or higher.

## What's Changed
* Closes #240: Overhaul icons by @dreng in https://github.com/mattieserver/netbox-topology-views/pull/272
* Fixes #253: Display device labels on top of logical connections by @dreng in https://github.com/mattieserver/netbox-topology-views/pull/273
* Fixes #274: Incorrect names displayed for power panels/feeds by @dreng in https://github.com/mattieserver/netbox-topology-views/pull/278
* Fixes #276: Delete RoleImages when their DeviceRole is deleted by @ManiacMaxo in https://github.com/mattieserver/netbox-topology-views/pull/279
* Fixes #281: fix imports for netbox v3.5 by @mattieserver in https://github.com/mattieserver/netbox-topology-views/pull/282
* bump version for v3.4 by @mattieserver in https://github.com/mattieserver/netbox-topology-views/pull/283

**Full Changelog**: https://github.com/mattieserver/netbox-topology-views/compare/v3.3.0...v3.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/mattieserver/netbox-topology-views/releases/tag/v3.4.0'>v3.4.0</a>.</em>

**Comment by dreng** [comment]
That was quick. Very cool!

> This version only works for netbox 3.5.0 or higher.

I just gave it a try in our test production server and I have had no problems at all upgrading just netbox-topology-views to v3.4.0 with NetBox v3.4.9. Creating topologies works just like it did before.
